src/schema/nmdc.yaml

```diff
@@ -2324,17 +2346,32 @@ slots:
   metagenome_annotation_id:
     domain: FunctionalAnnotationAggMember
     range: WorkflowExecutionActivity
+    required: true
+    description: The value for slot id for the analysis activity (WorkflowExecutionActivity) that generated functional annotation
   gene_function_id:
     range: uriorcurie
+    required: true
+    description: The identifier for the gene function, ie KEGG.ORTHOLOGY:K00627
```

Comment:
Assuming `KEGG.ORTHOLOGY:K00627` is an example value, I'd recommend:

```diff
- description: The identifier for the gene function, ie KEGG.ORTHOLOGY:K00627
+ description: The identifier for the gene function; e.g., KEGG.ORTHOLOGY:K00627
```

Reference: https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/

Reply:
Moved to use examples syntax.